### PR TITLE
Removes the special off Cult Daggers

### PR DESCRIPTION
**Pull request by Aurorablade**
CAUSE GOD YOU PEOPLE CANNOT BE TRUSTED WITH SOME THINGS.

Now deals 5 bleed on attack.
Adds a cultist wield check.
Propgating over to my cultpass PR.

:cl: Fethas
tweak: The cultist dagger has had its special removed, it still cuases small bleed and retains its embed chance.
/:cl:

### CODE REVIEW

**Comment by Aurorablade** [comment]
stop making emojis.

**Comment by FlattestGuitar** [comment]
🔪🔪🔪☠️🕯️👌🙃😯💦💦💦

**Comment by MarcellusPye** [comment]
👍 👎 💯 🔢 🥇 🥈 👆 🥉 🅰️ 🆎 🔤 🔡 🉑 ♿️ 🤧 🉐 🅱️ 👶 🍼 🐤 🚼

**Comment by uraniummeltdown** [comment]
How about bleed 50 instead of 5? It's not too low and it's not as crazy as bleed 5000 of the special attack

File: code/game/gamemodes/cult/cult_items.dm [hunk 1]
```diff
@@ -45,12 +45,21 @@
...
+			user.adjustBruteLoss(rand(force/2,force))
+
 	if(ishuman(target))
 		var/mob/living/carbon/human/H = target
-		if(!(cooldown > world.time) && ((H.stat != DEAD) && !(NO_BLOOD in H.species.species_traits)))
-			user.visible_message("<span class='danger'>The runes on the blade absorb the blood of [H]!</span>")
-			H.bleed(5000)
-			cooldown = world.time + 2400
+		if((H.stat != DEAD) && !(NO_BLOOD in H.species.species_traits))
+			H.bleed(5)
```
Comment (Fox-McCloud) [inline code/game/gamemodes/cult/cult_items.dm]: Why not just do what TG did, both in terms of preventing non-cultists from using it (in `pickup`) and damage?

**Comment by DesolateG** [comment]
Anything to get rid of the stupidity of two stabs being able to kill someone.

File: code/game/gamemodes/cult/cult_items.dm [hunk 2]
```diff
@@ -43,14 +43,12 @@
...
 
-/obj/item/weapon/melee/cultblade/dagger/afterattack(atom/target, mob/living/carbon/human/user)
+/obj/item/weapon/melee/cultblade/dagger/attack(atom/target, mob/living/carbon/human/user)
 	..()
 	if(ishuman(target))
 		var/mob/living/carbon/human/H = target
-		if(!(cooldown > world.time) && ((H.stat != DEAD) && !(NO_BLOOD in H.species.species_traits)))
-			user.visible_message("<span class='danger'>The runes on the blade absorb the blood of [H]!</span>")
-			H.bleed(5000)
-			cooldown = world.time + 2400
```
Comment (Crazylemon64) [inline code/game/gamemodes/cult/cult_items.dm]: The `cooldown` variable is unused now, could it be removed?

**Comment by DesolateG** [comment]
@Aurorablade as a cultist that regularly uses these I still support this PR. However with eldritch sharpening they'll still be a useful weapon after the nerf right?

**Comment by Fox-McCloud** [comment]
Good riddance.

**Pull request merged**